.github/workflows/build-electron-app.yaml

```diff
@@ -0,0 +1,71 @@
+name: Build and release Electron app
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v1
+
+      - name: Setup Node.js and NPM
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.9.1
+
+      - name: Run electron-builder
+        uses: samuelmeuli/action-electron-builder@v1
+        with:
+          package_root: electron
+          github_token: ${{ secrets.github_token }}
+
+      - name: Cache electron-builder files
+        uses: actions/cache@v3
+        with:
+          path: |
+            electron/out/*dmg*
+            electron/out/*AppImage*
+            electron/out/*exe*
+            electron/out/latest*
+          key: electron-builder-${{ runner.os }}
+
+  release:
+    needs: build
+
+    permissions:
+      contents: write
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
```

Comment:
Does this job needs to run on all three platforms? I thought it only publishes the files, build is already done